.cirrus.yml

```diff
@@ -205,6 +207,6 @@ macos_task:
   compile_script:
     - make V=1 && make V=1 examples && cd tests && make V=1
   test_script:
-    - sudo make V=1 test-ci
+    - sudo make V=1 test-ci ${tflags}
```

Comment:
The `make` command does not take these options as parameter. Instead you need to pass them as environment variables. Since it is an environment variable anyway in the `env:`, it should be enough to uppercase it there and not change the `test_script`.

Reply:
Please try to add the parameter `-E` to `sudo` to make it keep environment variables. Right not it seems TFLAGS is lost.

Reply:
This seems to work now.